config/config.proto

```diff
@@ -82,3 +83,7 @@ message ConfigUnion {
     ZoneConfig zone = 1;
   }
 }
+
+message SystemConfig {
+  repeated proto.KeyValue values = 1 [(gogoproto.nullable) = false];
```

Comment:
a map would buy nothing unless I had really expensive processing to build it. the encoding/decoding just turns everything into slices and back again while doing extra sorting.